docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -164,11 +164,45 @@ Scale up the coverage of treatment and prevention interventions to 90%.
 
 .. _5.1:
 
-5.1 Vivarium concept model 
---------------------------
+5.1 Vivarium concept model diagram
+----------------------------------
 
 .. image:: sam_concept_model_diagram.svg
 
+5.1.1 Cause Models
+~~~~~~~~~~~~~~~~~~
+
+* :ref:`Diarrheal Diseases (GBD 2019) <2019_cause_diarrhea>`
+
+* :ref:`Lower Respiratory Infections (GBD 2019) <2019_cause_lower_respiratory_infections>`
+
+* :ref:`Measles (GBD 2019) <2019_cause_measles>`
+
+5.1.2 Joint Cause-Risk Models
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* :ref:`Child Wasting / Protein Energy Malnutrition (GBD 2020) <2020_risk_exposure_wasting_state_exposure>`
+
+5.1.3 Risk Exposure Models
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Non-exclusive Breastfeeding
+
+* Discontinued Breastfeeding
+
+* :ref:`Child Stunting (GBD 2020) <2020_risk_exposure_child_stunting>`
+
+* Low Birthweight and Short Gestation (GBD 2019)
+
+5.1.4 Risk Effects Models
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* :ref:`Low Birthweight and Short Gestation Risk Effects (GBD 2019) <2019_risk_effect_lbwsg>`
+
```

Comment:
I'm wondering if we should add another section called 
1) "risk risk correlation" which is the LBWSG and stunting/wasting correlations and 
2) then another sections called "risk risk effect" which is the wasting to stunting arrow

Reply:
@Ninicorn I'll go ahead and add a section for the risk-risk correlation.

I think risk-risk effects can just go in the 'risk effects' section  - the template was designed to be general enough to include both risk-cause and risk-risk. The wasting-stunting arrow should be one section in the "wasting risk effects" document (i.e. one of the listed outcomes would be stunting).

Also, I didn't know where to put the arrow for the vicious cycle, but I figured we could just decide once we actually have documentation.

Actually, since this PR has already been approved, I think I'll just merge it first and create a new PR with the new sections and new list items for the missing components, so that you and others will have a chance to review whether the new pieces look good.